### PR TITLE
Upgrade dependencies to match quality checklist

### LINKED ISSUE #62: Code Exchange quality checklist
The purpose of this checklist is to assess the state of this repository, and bring it to an acceptable level for long-term support by completing the tasks listed below.

**This checklist is used internally but can be used as a guidance for others**

> Please check every part of this checklist for us to process your request. You can do this by marking them with [x] instead of [ ].

### Documentation

- [x] `LICENSE` file exists in repository

- [x] [Code of Conduct](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md) exists in repository

- [x] [CONTRIBUTING](https://github.com/twilio-labs/.github/blob/master/CONTRIBUTING.md) exists in repository 

- [x] No broken links in README.md

- [x] Build status exists at top of README.md

- [x] Set up steps are complete and accurate

- [x] CodeExchange repository is linked in README.md

### GitHub

- [x] Dependabot set up

- [x] Dependabot automerge set up

- [x] Uses GitHub Actions (required for Twilio sample) or other CI

### Code

- [x] Packages/dependencies are up to date

- [x] Code compiles against latest packages/dependencies

- [x] Matches [CodeExchange Guidelines](https://github.com/twilio-labs/code-exchange/tree/master/guidelines) to the best of ability

### Tests

- [x] Functionality tests have been created

- [x] All tests pass

### App

- [x] Confirmed sample behaves as expected (after dependencies have been updated)

- [x] This file has been copied into a wiki entry for the respective repository or added as a GitHub issue

### General

- [x] App is released under a permissive license like MIT or Apache-2

- [x] I've read and followed the contribution guidelines

- [x] I've read and implemented my template according to the respective coding guidelines for my project

- [x] I agree and adhere to the Code of Conduct

- [x] The template has been authored by me / or my organization and will be kept up-to-date. If the template will not longer be maintained, I'll notify this project.

### PR DESCRIPTION
**Pull request by rojasTob**
Resolves #62

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+# OR
+# twilio api:core:applications:create --friendly-name=voice-client-javascript
+TWILIO_TWIML_APP_SID=APXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+# Your REST API Key information
+# https://www.twilio.com/console/project/api-keys
+# OR
+# twilio api:core:keys:create --friendly-name=voice-client-javascript -o json
+# NOTE: Make sure to copy the secret, it'll will only be displayed once
+API_KEY=SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```
Comment (dkundel) [inline .env.example]: Can we turn these into `TWILIO_API_KEY` and `TWILIO_API_SECRET`?

File: package.json [hunk 2]
```diff
@@ -24,17 +24,17 @@
   "author": "Twilio Developer Education",
   "license": "MIT",
   "engines": {
-    "node": ">=8.x"
+    "node": "^12"
```
Comment (dkundel) [inline package.json]: I think since v10 is still LTS it's fair to make this compatible for 10

File: README.md [hunk 3]
```diff
@@ -3,20 +3,20 @@
...
-Twilio&nbsp;Phone&nbsp;# | A Twilio phone number in [E.164 format](https://en.wikipedia.org/wiki/E.164) - you can [get one here](https://www.twilio.com/console/phone-numbers/incoming)
+`TWILIO_ACCOUNT_SID` | Your primary Twilio account identifier - find this [in the console here](https://www.twilio.com/console).
+`TWILIO_TWIML_APP_SID` | The TwiML application with a voice URL configured to access your server running this app - create one [in the console here](https://www.twilio.com/console/voice/twiml/apps). Also, you will need to configure the Voice "REQUEST URL" on the TwiML app once you've got your server up and running.
+`TWILIO_CALLER_ID` | A Twilio phone number in [E.164 format](https://en.wikipedia.org/wiki/E.164) - you can [get one here](https://www.twilio.com/console/phone-numbers/incoming)
+`API_KEY` / `API_SECRET` | Your REST API Key information needed to create an [Access Token](https://www.twilio.com/docs/iam/access-tokens) - create [one here](https://www.twilio.com/console/project/api-keys).
 
 ## Setting Up The Node.js Application
```
Comment (dkundel) [inline README.md:22]: Can we add a step on how to clone the project in here? Ideally in all READMEs

**Review by dkundel — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (rojasTob) [reply package.json]: Hi @dkundel. Github Actions are running for both versions 10 and 12. What do you think if we keep the 12 version in the package file?

Comment (dkundel) [reply package.json]: Right now this number means that it has to be node 12 or newer and excludes 10.

Comment (dkundel) [reply package.json]: `>=10.x` would probably be the better one here

Comment (rojasTob) [reply package.json]: @dkundel, thanks for your reply. I've changed the node version.

**Review by dkundel — APPROVED** [review]

**Pull request merged**